### PR TITLE
Added aac on ithc environment.

### PR DESCRIPTION
**Pull request by smathangi**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by adusumillipraveen — COMMENTED** [review]
Have u considered new way of adding apps ? which reduces a lot of duplication.https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment.md#application

**Comment by smathangi** [comment]
> Have u considered new way of adding apps ? which reduces a lot of duplication.https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment.md#application

Ok, just pushed. Could you review again please.

File: CODEOWNERS [hunk 1]
```diff
@@ -63,12 +63,11 @@ k8s/aat/common/ccd/case-document-am-api.yaml @hmcts/cdm
 
 ##AAC - CDM
 k8s/namespaces/aac/ @hmcts/cdm
+k8s/namespaces/aac/manage-case-assignment/ @hmcts/cdm
```
Comment (adusumillipraveen) [inline CODEOWNERS:66]:
```suggestion
```

File: k8s/ithc/common/aac/manage-case-assignment.yaml [hunk 2]
```diff
@@ -22,6 +22,6 @@ spec:
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/aac/manage-case-assignment:prod-dc892e19
     global:
-      environment: aat
+      environment: ithc
```
Comment (adusumillipraveen) [inline k8s/ithc/common/aac/manage-case-assignment.yaml:25]: Is this file still needed

**Review by adusumillipraveen — CHANGES_REQUESTED** [review]

File: k8s/ithc/common/aac/manage-case-assignment.yaml (hunk 2 above)
Comment (smathangi) [reply k8s/ithc/common/aac/manage-case-assignment.yaml]: I can't see in my local (probably git move/rename issue). Deleted now.

**Review by adusumillipraveen — APPROVED** [review]

**Review by hemantt — APPROVED** [review]
